packages/cra-template-imodeljs/template.json

```diff
@@ -0,0 +1,67 @@
+{
+  "package": {
+    "dependencies": {
+      "@bentley/backend-itwin-client": "^2.7.4",
+      "@bentley/bentleyjs-core": "^2.7.4",
+      "@bentley/context-registry-client": "^2.7.4",
+      "@bentley/express-server": "^2.7.4",
+      "@bentley/frontend-authorization-client": "^2.7.4",
+      "@bentley/geometry-core": "^2.7.4",
+      "@bentley/icons-generic-webfont": "^1.0.27",
+      "@bentley/imodelhub-client": "^2.7.4",
+      "@bentley/imodeljs-backend": "^2.7.4",
+      "@bentley/imodeljs-common": "^2.7.4",
+      "@bentley/imodeljs-frontend": "^2.7.4",
+      "@bentley/imodeljs-i18n": "^2.7.4",
+      "@bentley/imodeljs-quantity": "^2.7.4",
+      "@bentley/itwin-client": "^2.7.4",
+      "@bentley/orbitgt-core": "^2.7.4",
+      "@bentley/product-settings-client": "^2.7.4",
+      "@bentley/rbac-client": "^2.7.4",
+      "@bentley/react-scripts": "3.4.6",
+      "@bentley/telemetry-client": "^2.7.4",
+      "@bentley/ui-abstract": "^2.7.4",
+      "@bentley/ui-components": "^2.7.4",
+      "@bentley/ui-core": "^2.7.4",
+      "@bentley/webgl-compatibility": "^2.7.4",
+	    "@bentley/ecschema-metadata": "^2.7.4",
```

Comment:
nit: can you remove the tab and align this?

Reply:
Done.